index.html

```diff
@@ -6827,15 +6622,401 @@ <h2>Examples</h2>
+  A detailed discussion of privacy (and security) considerations for the Web of Things,
+  including a threat model that can be adapted to various circumstances, is
+  presented in the informative document [[WOT-SECURITY-GUIDELINES]].  
+  This section discusses only privacy risks
+  and possible mitigations directly relevant to the WoT Thing Description.
+  </p>
+  <p>The use of a WoT Thing Description introduces  
+  the privacy risks given in the following sections.
+  After each risk, we suggest some possible mitigations.
+  </p>
+  <section id="sec-privacy-consideration-context">
+  <h5>Context Fetching</h5>
+        <p>
+          WoT Thing Descriptions can be evaluated with a 
+	  JSON-LD 1.1 processor [[json-ld11]],
+          which typically follows links to remote contexts (i.e., TD context
+          extensions, see <a href="#sec-context-extensions"></a>)
+          automatically, resulting in the transfer of files
+          without the explicit request of the <a>Consumer</a> for each one. 
+	  If remote
+          contexts are served by third parties, it may allow them to gather
+          usage patterns or similar information leading to disclosure
+	  of private information, or information that can be used to infer
+	  private information.
+          In the case of the WoT, an attacker can also observe the network
+          traffic produced by such fetches and can use the metadata
+          of the fetch, such as the destination IP address,
+          to infer information about the device, 
+	  especially if domain-specific vocabularies are used.  
+	  This is a risk even if the connection
+          is encrypted, and is related to DNS privacy leaks.
+	  See also <a href="#sec-security-consideration-context-tampering"></a>,
+	  which is a related security risk which can also be avoided with the
+	  following mitigations.
+      <dl>
+	<dt>Mitigation:</dt>
+	<dd>
+          Implementations on resource-constrained devices are expected
+          to perform raw JSON processing (as opposed to JSON-LD processing)
+	  and support only a limited set of domain-specific extensions.
+	  The set of supported extensions can be established by a 
+	  WoT Profile [[WOT-PROFILE]], for example.
+	  <span class="rfc2119-assertion" id="security-static-context">
+          Constrained implementations SHOULD use statically managed and vetted
+          versions of their supported context extensions. 
+          </span>
+	  <span class="rfc2119-assertion" id="security-remote-context">
+          Constrained implementations SHOULD NOT follow links to remote contexts.
+	  </span>
+	  In this case the URLs are used only to identify extensions,
+	  not to fetch their definitions.
+	  <span class="rfc2119-assertion" id="security-update-contexts">
+          Supported context extensions on constrained implementations MAY be managed
+          through secure software update mechanisms.
+	  </span>
+        </dd>
```

Comment:
Last one might be a SHOULD rather than a MAY.    Will consider what to say about "constrained" devices, did want to allow more general-purpose processing in some contexts (SHOULD is optional, also).  Maybe refine the definition of "constrained" a little (and "only if necessary" etc).

Reply:
I think I did (but will check) cross-reference a related privacy consideration.